AWS/connectors/s3/trigger_s3_cloudfront.py

```diff
@@ -7,6 +7,7 @@
 import pandas as pd
 import time
 import datetime
+import json
```

Comment:
For performance reason, I suggest using `orjson`:
```suggestion
import orjson
```